project/plugins.sbt

```diff
@@ -16,7 +16,7 @@ addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.33")
 // sbt-osgi 0.9.5 is available but breaks including jdk9-only classes
 // sbt-osgi 0.9.6 is available but breaks populating pekko-protobuf-v3
 addSbtPlugin("com.typesafe.sbt" % "sbt-osgi" % "0.9.4")
-addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.8.1")
```

Comment:
This old version of sbt-mima is why it wasn't correctly resolving Scala 3 artifacts, this has since been resolved in latest MiMA